demo/src/app/shared/api-docs/api-docs-config.component.ts

```diff
@@ -19,7 +19,7 @@ const CONFIG_SUFFIX_LENGTH = 'Config'.length;
 @Component({
 	selector: 'ngbd-api-docs-config',
 	standalone: true,
-	imports: [RouterLinkWithHref, NgbdApiDocsBadge, NgForOf, NgIf, NgbdFragment],
+	imports: [RouterLink, NgbdApiDocsBadge, NgForOf, NgIf, NgbdFragment],
```

Comment:
NgForOf should be replaced with NgFor (everywhere)

Reply:
Fixed with #4433